src/model/draft/common/mef-types.yang

```diff
@@ -43,11 +43,18 @@ module mef-types {
 
   revision YYYY-MM-DD {
     description
-      "Initial Version.";
+      "Revision to address bugs found in integration..";
     reference
       "Legato - Services YANG SCA (MEF XX)";
```

Comment:
Spec number is missing

Reply:
Will replace XX with 58 in the reference statement.